src/rpms.sh

```diff
@@ -497,8 +546,9 @@ rlCheckMakefileRequires() {
 
 =head3 rlAssertRequired
 
-Ensures that all Requires, specified in beakerlib-style beaker-wizard layout
-Makefile, are installed.
+Ensures that all requires and recommends specified in the metadata.yaml are installed.
+If no metadata.yaml is present a fall back to Makefile is done to check Requires
```

Comment:
"fall back" shouldn't have the space in it I believe :)

Reply:
fixed by https://github.com/beakerlib/beakerlib/pull/75/commits/2731beeff30a9dc9a748a026ceb1409b08797c5f